analyzers/tests/SonarAnalyzer.Test/TestCases/CastShouldNotBeDuplicated.cshtml

```diff
@@ -0,0 +1,10 @@
+﻿@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
```

Comment:
This is necessary for `asp-for` to be properly interpreted by the generator.

If it's not included, we get the equivalent of garbage from the razor generator, since it parses `asp-for` as a random tag that has no specific meaning.
If it is included, we get the thing on the right of `asp-for` to be bound inside a lambda on the `.g.cs` file, which is what we are trying to exclude from the rule.